pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -76,7 +76,7 @@ func (alloc *Action) pickUpQueuesAndJobs(queues *util.PriorityQueue, jobsMap map
 				continue
 			}
 		} else if job.IsPending() {
-			klog.V(4).Infof("Job <%s/%s> Queue <%s> status update from pending to inqueue, reason: no enqueue action is configured.",
```

Comment:
inqueue is a status, while enqueue is an action.

Reply:
sorry, I changed it back